src/components/TextMaskedInput/TextMaskedInput.js

```diff
@@ -95,45 +95,25 @@ export const TextMaskedInput = (props) => {
       : `${base} ${styles.TextInputStylable}`
   }
 
-  const getMaskedInputByType = (mask) => {
-    if (typeof mask === 'function') {
-      return (
-        <MaskedInput
-          value={value}
-          mask={mask}
-          type={restProps.type}
-          data-tid={restProps['data-tid']}
-          onChange={onChange}
-          onBlur={onBlur}
-          name={name}
-          placeholder={restProps.placeholder}
-          className={getClasses()}
-          disabled={restProps.disabled}
-          placeholderChar={placeholderChar}
-          autoComplete={autoComplete}
-        />
-      )
-    } else {
-      return (
-        <MaskedInput
-          value={value}
-          mask={mask}
-          type={restProps.type}
-          data-tid={restProps['data-tid']}
-          guide={restProps.guide}
-          onChange={onChange}
-          onBlur={onBlur}
-          name={name}
-          placeholder={restProps.placeholder}
-          className={getClasses()}
-          disabled={restProps.disabled}
-          keepCharPositions={restProps.keepCharPositions}
-          pipe={restProps.pipe}
-          placeholderChar={placeholderChar}
-          autoComplete={autoComplete}
-        />
-      )
-    }
+  const maskedInputProps = {
+    value,
+    mask,
+    type: restProps.type,
+    'data-tid': restProps['data-tid'],
+    onChange,
+    onBlur,
+    name,
+    placeholder: restProps.placeholder,
+    className: getClasses(),
+    disabled: restProps.disabled,
+    placeholderChar,
+    autoComplete,
+  }
+
+  if (typeof mask !== 'function') {
```

Comment:
It was tough to see the difference between the two branches, I hope this helps.

Reply:
Yeah that's a lot easier to read.